archetypes/quickstart-se/src/main/resources/src/test/resources/config-profile.yaml

```diff
@@ -0,0 +1,11 @@
+# Use a profile (without a name) to use a random port in tests
```

Comment:
`archetypes/quickstart-se` does not exist anymore, Templates and files are exclusively located under `archetypes/helidon/src/main/archetype`.

Reply:
Removed changes in archetypes